2-mandatory/1-writers.js

```diff
@@ -55,39 +55,39 @@ let writers = [
 
 /*
 Exercise 1:
-  Loop through the Array, and for each object, use `console.log()` to print out the below sentence
+  Loop through the Array, and for each object, use console.log() to print out the below sentence
   and insert the corresponding values to the place holders that are indicated in curly braces:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  return  writers.forEach(element => console.log(`Hi, my name is ${element.firstName} ${element.lastName}. I am ${element.age} years old, and work as a ${element.occupation}.`));
```

Comment:
nit: naming variables can be difficult, but in this case I think using `writer` instead of `element` would be better